test/e2e/testing_manifests/deferred/tuned-basic-updates-00.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: ocp-prof-deferred-updates-00
+  namespace: openshift-cluster-node-tuning-operator
+  annotations:
+    tuned.openshift.io/deferred: "update"
+    verificationCommand: "[\"/usr/sbin/sysctl\", \"-n\", \"vm.swappiness\"]"
+    verificationOutput: "17"
+spec:
+  profile:
+  - data: |
+      [main]
+      summary=Custom OpenShift profile
+      include=openshift-node
+      [sysctl]
+      vm.swappiness=17
```

Comment:
If we're checking only for `vm.swappiness`, is there a reason to set the following 3 sysctls too?  I understand this is just an example, but changing sysctls unnecessarily during e2e test might not be the greatest idea.  If you really want to keep them (as an example), it might be safer to comment them out.  Similarly in other manifests.

Reply:
removing them